sparql/sparql12/syntax-triple-terms-negative/list-tripleterm-01.rq

```diff
@@ -0,0 +1,9 @@
+PREFIX ex: <http://example.com/>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+
+SELECT * WHERE {
+    <<(?s ?p ?o)>> ex:author ex:Bob .
+    <<( ex:Moscow a ex:City )>> ex:author ?person .
+    # TODO: See if this should be throwing an error
+    <<( _:b ex:x () )>> ex:broken true .
```

Comment:
Same question here as with `list-anonreifier-01` – is this about `()`/`rdf:nil`?